### ISSUE TITLE
Site 118: Billing Discrepancies # 2

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#880]
In GitLab by @kdjstudios on May 7, 2018, 08:16

**Submitted by:** "Miya Harris" <miya.harris@answernet.com>

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/2018-05-04-21762/conversation

**Server:** Internal

**Client/Site:** Sarasota

**Account:** Freedom Medical

**Issue:** 

Good Afternoon All- I have another account with the exact same issue as the ticket I reported yesterday

 

Site: Sarasota

Account: Freedom Medical

Issue: Billing Discrepancies: Balance is incorrect

 

Details:

April 2014 invoice showed as 111.25 , however; the negative balance was not captured on the account. The previous balance on this invoice was -$41.30 leaving the invoice amount at $69.95

The clients balance is showing $410.46 however; in the Account Ledger it is showing $-23.09

I am also showing payment errors: Payment 8/03/2014 was a payment for $474.95 the ledger is showing the payments broken down by invoice (118-19931 $81.50 ,118-20843 $133.60,118.21761 $137.80, 118-23605 $122.05) AND the full payment

 

 

Summary:

If the $-41.30 was calculated on the first invoice and the $474.95 payment was removed (as it was already calculated) the amount would match

 

I already explained the error to the client however; the client sees the incorrect information and would like it to be corrected before making any payments. Please correct the issues so that we can provide the client with accurate information.

[FREEDOM+MEDICAL+-+Ledger+-+04May2018+1.csv](/uploads/88ead775503260f8cb2222d1c7541290/FREEDOM+MEDICAL+-+Ledger+-+04May2018+1.csv)

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 7, 2018, 08:59

changed the description

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 7, 2018, 09:49

Update from Miya:

> While we are waiting on a resolution. Is it possible to add a note on the “billing history” when clients are logging in to view their balance?
> 
> A message saying “ Attention we are performing maintenance updates. As a result, some balances are not accurate. Please contact your local office for your account balance”.
> 
> Something like this that will prompt them to call us for a true balance.

My response:

> I do not believe that SA Billing has any way to display such a message. However, we have informed the support team and will update you accordingly.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 7, 2018, 09:49

changed title from **Site 118: Billing Discrepancies #2** to **Site 118: Billing Discrepancies #{+ +}2**

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 23, 2018, 15:20

@tim.traylor @thomas.gentry @smasih @pchaudhary @rchauhan 

Just to confirm SA Billing does not have this type of functionality?

@tim.traylor 

I see the benefit in this and could also help us identify faulty accounts to operations. Should we get an estimate on creating such a feature?

**Comment by kdjstudios** [comment]
In GitLab by @smasih on May 24, 2018, 07:06

@kdjstudios 


> Just to confirm SA Billing does not have this type of functionality?

Unfortunately, we don't have any broadcast functionality like this.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 24, 2018, 13:21

Marking for discussion. So Gary can approve or disprove this type of functionality request and also set a priority. If so we will create a new ticket for that request.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 24, 2018, 13:23

@smasih 

Also any updates on the actual account issue mentioned in the ticket?

**Comment by kdjstudios** [comment]
In GitLab by @smasih on May 25, 2018, 01:36

@kdjstudios 

`Thu, 11 Sep 2014` is the conversion date of "Sarasota FL" so any data before that would be ignored as per   @tim.traylor.

So after removing transactions before and on conversion date we get right amount in Account ledger. (tested in merge)



**But, the issue over here is different:**

* Client knows about the historic invoices

* New account ledger is not released on client portal

We will discuss this with Tim during our morning huddle.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 29, 2018, 09:46

I will check the account ledger on client login with multiple accounts to see how it is displayed.

**Comment by kdjstudios** [comment]
In GitLab by @tim.traylor on May 29, 2018, 09:50

@smasih Sumeet, please go ahead and apply the conversion date filter to the old client portal ledger while we determine the best way to implement the new ledger for the portal.  Thx

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Jun 1, 2018, 07:59

@tim.traylor 

Tested the old ledger by supplying it site conversion date but still, it's not fixed.

It can be fixed but for that, we need to write extra code for it in the ledger.

I would suggest that we should release new ledger to the client portal instead of fixing the old one.

**Comment by kdjstudios** [comment]
In GitLab by @tim.traylor on Jun 4, 2018, 06:07

@smasih Confirming: if a customer has multiple accounts they should be listed in different sections of the new ledger, in account name and account number order. This would be as if you were to run the two accounts separately and then just concatenated the reports.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Jun 4, 2018, 23:11

assigned to @smasih

**Comment by kdjstudios** [comment]
In GitLab by @tim.traylor on Jun 15, 2018, 06:45

@smasih @kdjstudios We will need two customizations when the new ledger is being displayed to our end (or our external customer's) end customer:
The 'created at' column should be hidden.
The verbiage regarding not showing transactions prior to the conversion date needs to be different.  We will need Gary to provide or approve the wording.  We should review w/him on 6/22.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Jun 25, 2018, 04:33

moving to next milestone

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 15, 2018, 08:08

@tim.traylor @pchaudhary @smasih 

Miya closed this ticket in the HD. Closing here too.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 16, 2018, 11:44

closed
